data/osx/Cataclysm.sh

```diff
@@ -4,6 +4,7 @@ cd "${PWD}/../Resources/"
 
 COMMAND='
 OSREV=`uname -r | cut -d. -f1`;
+export TERMINFO=/usr/share/terminfo;
```

Comment:
I'm not sure, even before the apple script was introduced, I had already met this problem: `Error opening terminal: xterm-256color`.